components/Elevation/tests/unit/MaterialElevationColorTests.m

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialElevation.h"
+#import "MaterialMath.h"
+#import "UIColor+MaterialDynamic.h"
+
+@interface MaterialElevationColorTests : XCTestCase
+
+@end
+
+@implementation MaterialElevationColorTests
+
+- (void)testResolvedColorWithElevation {
+  // Given
+  CGFloat elevation = (CGFloat)10;
+  UIColor *color = UIColor.blackColor;
+
+  // When
+  UIColor *resolvedColor = [color resolvedColorWithElevation:elevation];
+  UIColor *expectedColor = [UIColor colorWithRed:(CGFloat)0.1279052872759267
+                                           green:(CGFloat)0.1279052872759267
+                                            blue:(CGFloat)0.1279052872759267
+                                           alpha:(CGFloat)1];
+
+  // Then
+  XCTAssertTrue([self compareColorsWithFloatPrecisionFirstColor:resolvedColor
```

Comment:
Nit: Please include a failure string that will make the error output useful. Here and throughout. Consider testing the string by causing an intentional failure while writing the test.

For example,

```objc
XCTAssertTrue([self compareColorsWithFloatPrecisionFirstColor:resolvedColor secondColor:expectedColor], @"(%@) is not equal to (%@)", resolvedColor, expectedColor);
```

Reply:
Addressed in ec78c1cf6e8932e4ac318e277dae7e31646bcab5